lib/FroalaEditorFunctionality.jsx

```diff
@@ -149,44 +155,45 @@ var FroalaEditorFunctionality = {
       this.$element = null;
       this.editorInitialized = false;
     }
-  },
+  }
 
-  getEditor: function() {
+  getEditor () {
 
     if (this.$element) {
       return this.$element.froalaEditor.bind(this.$element);
     }
+
     return null;
-  },
+  }
 
-  generateManualController: function() {
+  generateManualController () {
+    let self = this;
 
-    var self = this;
-    var controls = {
-      initialize: this.createEditor,
-      destroy: this.destroyEditor,
-      getEditor: this.getEditor,
-    };
+    let controls = Object.assign({
```

Comment:
Why use the `Object.assign`-method when you only have 1 object?
You can just use a basic object I think?